### PR TITLE
fix(forge): add logs/decoded logs in json test results

### LINKED ISSUE #9064: Provide `decoded_logs`  in `forge test --json`
### Component

Forge

### Describe the feature you would like

It seems `forge test --json` no longer includes the `decoded_logs` as it used to. Only `logs`, such as:

```
"logs":[{
  "address": "0x000000000000000000636F6e736F6c652e6c6f67",
  "topics": [
    "0x41304facd9323d75b11bcdd609cb38effffdb05710f7caf0e9b16c6d9d709f50"
  ],
  "data": "0x00000000000000000000000000000000000000000000000000000000000000200000000000000000000000000000000000000000000000000000000000000012436f6e74726163742073697a653a203133380000000000000000000000000000"
}]
```

### Additional context

According to the discussion in an old issue (#3002), this was feasible and simple. Did anything change? Is this no longer possible because of some technical reason I am not aware of?

### PR DESCRIPTION
**Pull request by grandizzy**
## Motivation
Closes #9064 

## Solution

- add logs/decoded logs in json response (verbosity based)

### CODE REVIEW

**Review by zerosnacks — APPROVED** [review]
lgtm :+1:

File: crates/forge/tests/cli/test_cmd.rs [hunk 1]
```diff
@@ -275,7 +275,10 @@ Ran 1 test suite [ELAPSED]: 1 tests passed, 0 failed, 0 skipped (1 total tests)
 });
 
 const SIMPLE_CONTRACT: &str = r#"
+pragma solidity 0.8.18;
```
Comment (DaniPopes) [inline crates/forge/tests/cli/test_cmd.rs:278]: pls dont add pragma, its done in test project

Comment (grandizzy) [reply crates/forge/tests/cli/test_cmd.rs:278]: noted, thing here is that by importing console.sol test failed with
```
Found incompatible versions:
src/Simple.t.sol =0.8.27 imports:
    src/test.sol >=0.4.23
    src/console.sol =0.8.18

```
hence the quick fix

Comment (DaniPopes) [reply crates/forge/tests/cli/test_cmd.rs:278]: I see, it should be relaxed in console.sol

Comment (DaniPopes) [reply crates/forge/tests/cli/test_cmd.rs:278]: https://github.com/foundry-rs/foundry/pull/9078

**Pull request merged**